### PR TITLE
[Docs] Added docs on the newly supported builtin string functions

### PR DESCRIPTION
**Pull request by darionyaphet**
Append string functions description both English and Chinese document.

### CODE REVIEW

File: docs/manual-EN/functions-and-operators/functions-and-operator-reference.md [hunk 1]
```diff
@@ -1,24 +1,35 @@
...
-|  sqrt()     |  Return the square root of the argument    | 
-|  strcasecmp()  | Compare strings without sensitivity to case   | 
-|  tan()  |  Return the tangent of the argument | 
+|  ***Name***        |  ***Description***                                                        |
+|:-------------|:-------------------------------------------------------------------:|
+|  abs         |  Return the absolute value                                          |
+|  acos        |  Return the arc cosine                                              |
+|  asin        |  Return the arc sine                                                |
+|  atan        |  Return the arc tangent                                             |
+|  cbrt        |   Returns the cubic root of the argument                            |
```
Comment (Amber1990Zhang) [inline docs/manual-EN/functions-and-operators/functions-and-operator-reference.md]: Returns >- Return

File: docs/manual-EN/functions-and-operators/functions-and-operator-reference.md [hunk 2]
```diff
@@ -1,24 +1,35 @@
...
+|  ceil        |  Return the smallest integer value not less than the argument       |
+|  cos         |  Return the cosine                                                  |
+|  exp         |   Raise to the power of                                             |
+|  floor       |  Return the largest integer value not greater than the argument     |
+|  hypot       |  Returns the hypotenuse of a right-angled triangle                  |
```
Comment (Amber1990Zhang) [inline docs/manual-EN/functions-and-operators/functions-and-operator-reference.md]: Ditto, please unify the `Return` format.

File: docs/manual-EN/functions-and-operators/functions-and-operator-reference.md [hunk 3]
```diff
@@ -1,24 +1,35 @@
...
+|  length      |  Return the length of a string in bytes                             |
+|  left        |  Return the leftmost number of characters as specified              |
+|  log         |   Return the natural logarithm of the first argument                |
```
Comment (Amber1990Zhang) [inline docs/manual-EN/functions-and-operators/functions-and-operator-reference.md]: Why `the first argument`? Is the function supposed to assign multiple values? Please check and unify the expression.

File: docs/manual-EN/functions-and-operators/functions-and-operator-reference.md [hunk 4]
```diff
@@ -1,24 +1,35 @@
...
+|  lower       |  Return the argument in lowercase                                   |
+|  lpad        |  Return the string argument, left-padded with the specified string  |
+|  ltrim       |  Remove leading spaces                                              |
+|  now         |  Return the current date and time                                   |
+|  pow         |   Return the argument raised to the specified power                 |
+|  rand32      |  Return a random 32 bit interger                                    |
+|  rand64      |  Return a random 64 bit interger                                    |
+|  right       |  Return the specified rightmost number of characters                |
+|  round       |  Round the argument                                                 |
+|  rpad        |  Append string the specified number of times                        |
```
Comment (Amber1990Zhang) [inline docs/manual-EN/functions-and-operators/functions-and-operator-reference.md]: Pad the right side of a string with a specific set of characters

File: docs/manual-EN/functions-and-operators/functions-and-operator-reference.md [hunk 5]
```diff
@@ -1,24 +1,35 @@
...
+|  exp         |   Raise to the power of                                             |
+|  floor       |  Return the largest integer value not greater than the argument     |
+|  hypot       |  Returns the hypotenuse of a right-angled triangle                  |
+|  length      |  Return the length of a string in bytes                             |
+|  left        |  Return the leftmost number of characters as specified              |
+|  log         |   Return the natural logarithm of the first argument                |
+|  log2        |  Return the base-2 logarithm of the argument                        |
+|  log10       |  Return the base-10 logarithm of the argument                       |
+|  lower       |  Return the argument in lowercase                                   |
+|  lpad        |  Return the string argument, left-padded with the specified string  |
```
Comment (Amber1990Zhang) [inline docs/manual-EN/functions-and-operators/functions-and-operator-reference.md]: Pad the left side of a string with a specific set of characters

File: docs/manual-EN/functions-and-operators/functions-and-operator-reference.md [hunk 6]
```diff
@@ -1,24 +1,35 @@
...
+|  ltrim       |  Remove leading spaces                                              |
+|  now         |  Return the current date and time                                   |
+|  pow         |   Return the argument raised to the specified power                 |
+|  rand32      |  Return a random 32 bit interger                                    |
+|  rand64      |  Return a random 64 bit interger                                    |
+|  right       |  Return the specified rightmost number of characters                |
+|  round       |  Round the argument                                                 |
+|  rpad        |  Append string the specified number of times                        |
+|  rtrim       |  Remove trailing spaces                                             |
```
Comment (Amber1990Zhang) [inline docs/manual-EN/functions-and-operators/functions-and-operator-reference.md]: Return a string after truncating all trailing blanks

File: docs/manual-EN/functions-and-operators/functions-and-operator-reference.md [hunk 7]
```diff
@@ -1,24 +1,35 @@
...
+|  sin         |  Return the sine of the argument                                    |
+|  sqrt        |  Return the square root of the argument                             |
+|  strcasecmp  | Compare strings without sensitivity to case                         |
```
Comment (Amber1990Zhang) [inline docs/manual-EN/functions-and-operators/functions-and-operator-reference.md]: Compare strings without case sensitivity

File: docs/manual-EN/functions-and-operators/functions-and-operator-reference.md (hunk 4 above)
Comment (darionyaphet) [reply docs/manual-EN/functions-and-operators/functions-and-operator-reference.md]: This is reference from `MySQL` doc

**Comment by Amber1990Zhang** [comment]
LGTM

**Pull request merged**